packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-logs/job-logs.html

```diff
@@ -0,0 +1,40 @@
+<div data-ui-view>
+    <div class="logger col-md-12" data-scroll-glue>
+        <div
+            data-ng-if="$ctrl.logger.logs.logsAddress === null && !$ctrl.isJobTerminated()"
+        >
+            <div
+                class="logger__entry"
+                ng-repeat="entry in $ctrl.logger.logs.logs track by entry.id"
+            >
+                <span class="logger__entry-message"
+                    >{{($ctrl.moment(entry.id /
+                    1000000.).toISOString()).padEnd(25,' ') }} {{ entry.content
```

Comment:
It looks like we are doing a lot of things in the view. Could this be done by a formatter somewhere else